Resources/Prototypes/Guidebook/chemicals.yml

```diff
@@ -0,0 +1,60 @@
+- type: guideEntry
+  id: Chemicals
+  name: guide-entry-chemicals
+  text: "/ServerInfo/Guidebook/Chemicals.xml"
+  children:
+  - Elements
+  - Medicine
+  - Narcotics
+  - Pyrotechnic
+  - Toxins
+  - Foods
+  - Botanical
+  - Biological
+  - Others
+  filterEnabled: True
+
+- type: guideEntry
+  id: Elements
+  name: guide-entry-elements
+  text: "/ServerInfo/Guidebook/Chemical Tabs/Elements.xml"
```

Comment:
Consider changing the `Chemical Tabs` directory to `ChemicalTabs` or even `Chemicals`. Spaces are not great in file paths.